doc/api/fs.md

```diff
@@ -3201,6 +3208,12 @@ passed as the first argument. In this case, `fs.ftruncateSync()` is called.
 Passing a file descriptor is deprecated and may result in an error being thrown
 in the future.
 
+Here is an example below:
+
+```js
+fs.truncateSync('./foo', 10);
```

Comment:
This example doesn't provide any information that isn't in the function signature. Can we make this example more meaningful?